Gemfile

```diff
@@ -37,7 +37,7 @@ gem 'jquery-rails', '~> 4.3.1'
 gem "font-awesome-rails", '~> 4.7.0'
 
 # Security headers
-gem 'secure_headers', '~>3.6.7'
+gem 'secure_headers', '~>3.9.0'
```

Comment:
Looks safe. See change log below. `3.9.0` seems empty.

## 3.8.0

Fixes semicolon injection issue reported by @mvgijssel see https://github.com/twitter/secure_headers/issues/418

## 3.7.4

Backport SameSite=None functionality into 3.x line

## 3.7.3

- Updates `Expect-CT` header to use a comma separator between directives, as specified in the most current spec.

## 3.7.2

- Adds support for `worker-src` CSP directive to 3.x line (https://github.com/twitter/secureheaders/pull/364)

## 3.7.1

Fix support for the sandbox attribute of CSP. `true` and `[]` represent the maximally restricted policy (`sandbox;`) and validate other values.

## 3.7.0

Adds support for the `Expect-CT` header (@jacobbednarz: https://github.com/twitter/secureheaders/pull/322)